rnd/autogpt_builder/src/components/SchemaTooltip.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "@/components/ui/tooltip"
+import { BlockSchema } from "@/lib/types";
+import { Info } from 'lucide-react';
+import ReactMarkdown from 'react-markdown';
+
+type Props = {
+  schema: BlockSchema;
+}
+
+const SchemaTooltip: React.FC<Props> = ({ schema }) => {
```

Comment:
one of the reasons I like TypeScript is because you can do: 
```suggestion
const SchemaTooltip: React.FC<{ schema: BlockSchema }> = ({ schema }) => {
```